notebooks/a01_manifold_class.ipynb

```diff
@@ -0,0 +1,1630 @@
+{
```

Comment:
<p>Maybe add:</p><p> </p><p>We can indeed verify in the hierarchy of classes that the Hypersphere is implemented as a LevelSet, which we can see <a href="https://github.com/geomstats/geomstats/blob/20cccf598e94823ef8d7c0821b2968e6d29660c0/geomstats/geometry/hypersphere.py#L22" target="_blank">by also looking at the line</a> <code>class _Hypersphere(LevelSet)</code>  .</p><p> </p><p>The lines:</p><pre class="ql-syntax" spellcheck="false">        super(_Hypersphere, self).__init__(
	            dim=dim,
	            embedding_space=Euclidean(dim + 1),
	            submersion=lambda x: gs.sum(x**2, axis=-1),
	            value=1.0,
</pre><p> </p><p>precisely define the function f (called "submersion") and allow to build an hypersphere as the levelset for the level (called "value") 1.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1340/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>

Reply:
<p>i added the line:</p><p>You can see&nbsp;<a href="https://github.com/geomstats/geomstats/blob/20cccf598e94823ef8d7c0821b2968e6d29660c0/geomstats/geometry/hypersphere.py#L22" target="_blank">here</a>&nbsp;that the subclass&nbsp;<code>Hypersphere</code>&nbsp;is indeed implemented as a&nbsp;<code>LevelSet</code></p>